### PR TITLE
Allow theming text area handle

### PR DESCRIPTION
**Pull request by timja**
See https://github.com/jenkinsci/dark-theme-plugin/issues/103
and https://github.com/jenkinsci/dark-theme-plugin/pull/117

### Proposed changelog entries

* Entry 1: Developer: Allow theming text area handle
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by res0nance — APPROVED** [review]

File: war/src/main/less/base/style.less [hunk 1]
```diff
@@ -1332,11 +1332,12 @@ DIV.to-be-removed { display: none; }
...
     margin-bottom: 0;
 }
-DIV.textarea-handle {
+
+div.textarea-handle {
     height: 5px;
     font-size: 0;
-    background: #EEE url(../../images/textarea-handle.gif) no-repeat 50% 1px;
-    border: 1px solid #BABDB6;
+    background: var(--text-area-handle-bg-color) url(../../images/textarea-handle.gif) no-repeat 50% 1px;
```
Comment (oleg-nenashev) [inline war/src/main/less/base/style.less:1339]: Does it require a CSS fallback in this case?

Comment (timja) [reply war/src/main/less/base/style.less:1339]: No only in css files, that aren’t processed by webpack, less files are fine

**Review by oleg-nenashev — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, we'll merge it in 24 hours if there's no negative feedback

**Comment by fqueiruga** [comment]
I think it would be better to use the `--medium-grey` and `--light-grey` variables as defined on https://github.com/jenkinsci/jenkins/pull/4835 instead of defining two new variables here.

**Comment by timja** [comment]
> I think it would be better to use the `--medium-grey` and `--light-grey` variables as defined on #4835 instead of defining two new variables here.

pushed that awhile ago but github wasn't showing it, added a new commit and it's now there

**Review by fqueiruga — APPROVED** [review]

**Pull request merged**